prereise/gather/griddata/hifld/const.py

```diff
@@ -118,3 +118,67 @@
     "345": 345,
     "500": 500,
 }
+
+line_reactance_per_mile = {  # per-unit
+    69: 0.0096,
+    100: 0.0063,
+    115: 0.0039,
+    138: 0.0026,
+    161: 0.0021,
+    230: 0.0011,
+    345: 0.0005,
+    500: 0.0002,
+    765: 0.0001,
+}
+line_rating_short = {  # MVA
+    69: 86,
+    100: 181,
+    115: 239,
+    138: 382,
+    161: 446,
+    230: 797,
+    345: 1793,
+    500: 2598,
+    765: 5300,
+}
+line_rating_short_threshold = 50  # miles
+line_rating_surge_impedance_loading = {  # MVA
+    69: 13,
+    100: 30,
+    115: 35,
+    138: 50,
+    161: 69,
+    230: 140,
+    345: 375,
+    500: 1000,
+    765: 2250,
+}
+line_rating_surge_impedance_coefficient = 43.261
+line_rating_surge_impedance_exponent = -0.6678
+
+transformer_reactance = {  # per-unit
+    (69, 115): 0.14242,
+    (69, 138): 0.10895,
+    (69, 161): 0.14943,
+    (69, 230): 0.09538,
+    (69, 345): 0.08896,
+    (115, 161): 0.04516,
+    (115, 230): 0.04299,
+    (115, 345): 0.04020,
+    (115, 500): 0.06182,
+    (138, 161): 0.02818,
+    (138, 230): 0.03679,
+    (138, 345): 0.03889,
+    (138, 500): 0.03279,
+    (138, 765): 0.02284,
+    (161, 230): 0.06539,
+    (161, 345): 0.03293,
+    (161, 500): 0.06978,
+    (230, 345): 0.02085,
+    (230, 500): 0.01846,
+    (230, 765): 0.01616,
+    (345, 500): 0.01974,
+    (345, 765): 0.01625,
+    (500, 765): 0.00436,
+}
```

Comment:
Why not all the combinations are in the dictionary? Like 69-100, 161-765, etc.

Reply:
These came from an external document, we're working on validating/refining this table (see https://github.com/Breakthrough-Energy/PreREISE/pull/219#issuecomment-920286423). If anything combination is missing, the 'nearest' one should be chosen automatically.

Reply:
Sorry, I did not really follow the discussion